docs. update hotsync section (#548)
NethServer/docs: administrator-manual/en/hotsync.rst

   For a correct restore, it's suggested to configure HotSync on two identical servers or two servers with same network cards number, name and position. If the master and slave servers differ, the restore procedure may behave unexpectedly (see :ref:`hostync-troubleshooting-section`).

.. hint::

>    If you have configured slave host before November 2020, update the package to release 2.2.0 on master and slave with *yum update -y nethserver-hotsync --enablerepo=nethforge* or from Software Center.

Reviewer: Please do not add a date, just name the old version.

.. hint::

>    If you have configured slave host before release 2.2.0, update to this one (or superior) on master and slave with *yum update -y nethserver-hotsync --enablerepo=nethforge* or from Software Center.

Reviewer: ```suggestion
   If you have configured slave host before release 2.2.0, update to a newer one on master and slave with *yum update -y nethserver-hotsync --enablerepo=nethforge* or from Software Center.
```
